aperturedb/Configuration.py

```diff
@@ -14,6 +14,10 @@ class Configuration:
     use_ssl: bool = True
     use_rest: bool = False
     use_keepalive: bool = True
+    retry_connect_interval_seconds: int = 10
+    # Max number of attempts to retry the initial connection (0 means infinite)
+    # This is useful when the aperturedb server is not ready yet.
+    retry_connect_max_attempts: int = 0
```

Comment:
lets not do infinite as the default. I think expect a library to return an error at some point.

Reply:
I thought so too myself, but was convinced otherwise thinking of attaining analogous behavior and a current customer (watchdog).

1. So ConnectorRest does not encapsulate a connection. Not doing like this would introduce 2 separate default behaviors using rest or not.
2. The watchdog .... with increasing number of URLs, it'll be hard to ensure that we start it after all servers are up, and this is the case that got @luisremis because they anyways did not care (or write code to handle the response after creating the connector. I think this behavior can be explicitly documented but it gives the connectors a parity on how they init.

If you are convinced then, I can update the docs to highlight this, otherwise lets discuss this more.

Reply:
OK, I checked ConnectorRest and I stand corrected. Even that tries to connect in init, and bails out if server is not up.
I shall make this from infinite to 3, and 1 second for retry interval as default. 
Thanks for pointing this out.

Reply:
good to know!

We should put this in docs so we know our theory of operation on this stuff.